services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -481,31 +539,41 @@ impl NetCdfCfDataProvider {
             rasterband_channel: 0, // we calculate offsets below
             geo_transform,
             file_not_found_handling: FileNotFoundHandling::Error,
-            no_data_value: data_array.no_data_value(), // we could also leave this empty. The gdal source will try to get the correct one.
+            no_data_value: data_array.no_data_value_as_double(), // we could also leave this empty. The gdal source will try to get the correct one.
             properties_mapping: None,
-            width: dimensions.lon,
-            height: dimensions.lat,
+            width: dimensions
+                .get(3 /* 3 is lon */)
```

Comment:
maybe introduce a named constant for this

Reply:
[79e0669](https://github.com/geo-engine/geoengine/pull/605/commits/79e066964fccc9e97d4f2ad4a93c3a8d9e67b527)